dev-requirements.txt

```diff
@@ -1,35 +1 @@
-#
-# This file is autogenerated by pip-compile
-# To update, run:
-#
-#    pip-compile --output-file dev-requirements.txt dev-requirements.in
-#
-astroid==1.5.2            # via edx-lint, pylint, pylint-celery, pylint-plugin-utils
-backports.functools-lru-cache==1.3  # via astroid, pylint
-click-log==0.1.8          # via edx-lint
-click==6.7                # via click-log, edx-lint, pip-tools
-configparser==3.5.0       # via pylint
-cookies==2.2.1            # via responses
-edx-lint==0.5.4
-enum34==1.1.6             # via astroid
-first==2.0.1              # via pip-tools
-funcsigs==1.0.2           # via mock
-isort==4.2.5              # via pylint
-lazy-object-proxy==1.2.2  # via astroid
-mccabe==0.6.1             # via pylint
-mock==2.0.0               # via pytest-mock, responses
-pbr==3.0.0                # via mock
-pep8==1.7.0
-pip-tools==1.9.0
-py==1.4.33                # via pytest
-pylint-celery==0.3        # via edx-lint
-pylint-django==0.7.2      # via edx-lint
-pylint-plugin-utils==0.2.6  # via pylint-celery, pylint-django
-pylint==1.7.1
-pytest-mock==1.6.0
-pytest==3.0.7             # via pytest-mock
-requests==2.13.0          # via responses
-responses==0.5.1
-singledispatch==3.4.0.3   # via astroid, pylint
-six==1.10.0               # via astroid, edx-lint, mock, pip-tools, pylint, responses, singledispatch
-wrapt==1.10.10            # via astroid
+-r requirements/development.txt
```

Comment:
Maybe add a brief comment explaining the plan to remove this and the criteria for doing so?  Ditto for requirements.txt.